src/components/carousel/index.jsx

```diff
@@ -0,0 +1,21 @@
+import PropTypes from "prop-types";
+import Slider from "react-slick";
+
+const COMPONENT_CLASS_NAME = "c-carousel";
+
+const Carousel = ({ children, className, sliderProps }) => (
+	<div className={className ? `${COMPONENT_CLASS_NAME} ${className}` : `${COMPONENT_CLASS_NAME}`}>
```

Comment:
Do we think we need the wrapping element? or could we use the `className` prop of the `Slider` component?

Reply:
has this been addressed? @visionbegin @ecarlisle

Reply:
Seems to work with:
```
const Carousel = ({ children, className, sliderProps }) => (
	<Slider
		{...sliderProps}
		className={className ? `${COMPONENT_CLASS_NAME} ${className}` : `${COMPONENT_CLASS_NAME}`}
	>
		{children}
	</Slider>
);
```

Reply:
I do what the div wrap for flexibility.  Both the carousel and the internal slider has a className prop and want them to be treated independently.

Reply:
what flexibility did you imagine or want? Curious because I'd imagine the slider component itself would have a div around it or something that could achieve the same effect

Reply:
Not sure on the flexibility argument ATM, but applying the `className` to the `<Slider>` as I demoed would require merging the `className` with the `<Slider>` `className` coming on with `sliderProps` property (as documented by Slick) in order to not break things so that's potentially a bit more complicated here.  If we ignore that, the user may set the `className` following the Slick documents and our `className` would completely obliterate it.

Reply:
@visionbegin has this been addressed?

Reply:
AFAIK this has been resolved in that we're ok w/ the wrapping `<div>`.